From 4d49221b1fe92bf8b8b7d153f0f8b3d0db342443 Mon Sep 17 00:00:00 2001
From: "haotian.qu" <haotian.qu@amlogic.com>
Date: Tue, 16 May 2023 13:43:38 +0800
Subject: [PATCH 4/8] SM1: Rename SM1's config file. [1/1]

PD#SWPL-123893

Problem:
Uniform naming, the config file of sm1 should plus 'weston'

Solution:
Uniform naming, the config file of sm1 should plus 'weston'.

Verify:
local

Signed-off-by: haotian.qu <haotian.qu@amlogic.com>
Change-Id: Ie0aaf437e17dad048e24476f755cda193815488d
---
 conf/machine/mesonsm1-ac200-5.15.conf         | 87 +------------------
 .../machine/mesonsm1-ac200-weston-5.15.config | 86 ++++++++++++++++++
 2 files changed, 87 insertions(+), 86 deletions(-)
 mode change 100644 => 120000 conf/machine/mesonsm1-ac200-5.15.conf
 create mode 100644 conf/machine/mesonsm1-ac200-weston-5.15.config

diff --git a/conf/machine/mesonsm1-ac200-5.15.conf b/conf/machine/mesonsm1-ac200-5.15.conf
deleted file mode 100644
index 7c4ea5d0..00000000
--- a/conf/machine/mesonsm1-ac200-5.15.conf
+++ /dev/null
@@ -1,86 +0,0 @@
-#@TYPE: Machine
-#@NAME: meson
-
-#@DESCRIPTION: Machine configuration for meson systems
-
-include conf/machine/include/mesonsm1_k5.15_64b.inc
-include conf/machine/include/amlogic_weston.inc
-
-PREFERRED_PROVIDER_virtual/mesa = "libgles-eabihf-dvalin-wayland-drm"
-PREFERRED_PROVIDER_virtual/mesa-gl = "libgles-eabihf-dvalin-wayland-drm"
-PREFERRED_PROVIDER_virtual/libgbm = "libgles-eabihf-dvalin-wayland-drm"
-PREFERRED_PROVIDER_virtual/egl = "libgles-eabihf-dvalin-wayland-drm"
-PREFERRED_PROVIDER_virtual/libgl = "libgles-eabihf-dvalin-wayland-drm"
-PREFERRED_PROVIDER_virtual/libgles1 = "libgles-eabihf-dvalin-wayland-drm"
-PREFERRED_PROVIDER_virtual/libgles2 = "libgles-eabihf-dvalin-wayland-drm"
-
-MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " \
-        gpu \
-        wififw-ap6398 \
-        ap6xxx \
-        npu \
-        "
-
-RDEPENDS_packagegroup-amlogic-baserootfs_remove = " aml-subtitleserver"
-RDEPENDS_packagegroup-amlogic-baserootfs_remove = " gst-aml-drm-plugins"
-RDEPENDS_packagegroup-amlogic-baserootfs_remove = " fuse-exfat"
-RDEPENDS_packagegroup-amlogic-baserootfs_append = " npu-common"
-
-
-CHIPSET_NAME = "S905D3"
-KERNEL_DEVICETREE = "sm1_s905d3_ac200_linux.dtb"
-UBOOT_MACHINE = " sm1_ac200_v1_config"
-KERNEL_MODULE_AUTOLOAD += " dhd"
-KERNEL_MODULE_PROBECONF += "dhd"
-module_conf_dhd = "options dhd firmware_path=../../etc/wifi/6398/ nvram_path=../../etc/wifi/6398/"
-KERNEL_MODULE_AUTOLOAD += " snd-soc-dummy_codec"
-KERNEL_MODULE_AUTOLOAD += " snd-soc-aml_t9015"
-KERNEL_MODULE_AUTOLOAD += " snd-soc-tas5707"
-KERNEL_MODULE_AUTOLOAD += " snd-soc-aml_codec_tl1_acodec"
-KERNEL_MODULE_AUTOLOAD += " meson_ir"
-KERNEL_MODULE_AUTOLOAD += " hci_uart"
-KERNEL_MODULE_PROBECONF += "hci_uart"
-KERNEL_MODULE_AUTOLOAD += "meson-cpufreq"
-
-DISTRO_FEATURES_append = " drm"
-DISTRO_FEATURES_append = " gpt-partition"
-DISTRO_FEATURES_append = " kernel_515"
-
-#DISTRO_FEATURES_append = " onepass"
-DISTRO_FEATURES_append = " secure-u-boot optee "
-#DISTRO_FEATURES_append = " widevine "
-#DISTRO_FEATURES_remove = " alsa "
-DISTRO_FEATURES_append = " alsa "
-DISTRO_FEATURES_append = " pam"
-DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
-DEFAULT_DISTRO_FEATURES += " systemd"
-DISTRO_FEATURES_append = " systemd"
-DISTRO_FEATURES_remove = " sysvinit"
-DISTRO_FEATURES_append = " gstreamer1"
-DISTRO_FEATURES_append = " aml-dtv "
-DISTRO_FEATURES_append = " gst-plugin-venc"
-DISTRO_FEATURES_append = " aml-libjpeg"
-DISTRO_FEATURES_append = " aml-libvpcodec"
-DISTRO_FEATURES_append = " aml-libvphevcodec"
-#DISTRO_FEATURES_append = " playready "
-DISTRO_FEATURES_append = " softap"
-DISTRO_FEATURES_append = " aml-libgdc"
-#DISTRO_FEATURES_append = " swupdate"
-#DISTRO_FEATURES_append = " recovery"
-DISTRO_FEATURES_append = " gst-aml-imgproc"
-DISTRO_FEATURES_append = " gst-aml-gfx2d"
-DISTRO_FEATURES_append = " gst-aml-ion-allocator"
-DISTRO_FEATURES_append = " gst-jpeg-swdec"
-
-DISTRO_FEATURES_append = " gst-plugin-overlay"
-DISTRO_FEATURES_append = " gst-plugin-nn"
-DISTRO_FEATURES_append = " detect-library"
-DISTRO_FEATURES_append = " gst-plugin-conv"
-OVERRIDES .= ":ac200:sm1:onepass"
-MACHINEOVERRIDES .= ":client"
-
-EXTRA_BLUETOOTH_STUFF := "${@bb.utils.contains('DISTRO_FEATURES', 'bluetooth', bb.utils.contains('DISTRO_FEATURES', 'bluez5', ' bluez5', ' bluez4', d), '', d)}"
-EXTRA_BLUETOOTH_STUFF += " bluez-alsa"
-EXTRA_BLUETOOTH_STUFF += " brcm-patchram-plus"
-DISTRO_FEATURES_append = "${EXTRA_BLUETOOTH_STUFF}"
-
diff --git a/conf/machine/mesonsm1-ac200-5.15.conf b/conf/machine/mesonsm1-ac200-5.15.conf
new file mode 120000
index 00000000..af86cf1e
--- /dev/null
+++ b/conf/machine/mesonsm1-ac200-5.15.conf
@@ -0,0 +1 @@
+mesonsm1-ac200-weston-5.15.config
\ No newline at end of file
diff --git a/conf/machine/mesonsm1-ac200-weston-5.15.config b/conf/machine/mesonsm1-ac200-weston-5.15.config
new file mode 100644
index 00000000..7c4ea5d0
--- /dev/null
+++ b/conf/machine/mesonsm1-ac200-weston-5.15.config
@@ -0,0 +1,86 @@
+#@TYPE: Machine
+#@NAME: meson
+
+#@DESCRIPTION: Machine configuration for meson systems
+
+include conf/machine/include/mesonsm1_k5.15_64b.inc
+include conf/machine/include/amlogic_weston.inc
+
+PREFERRED_PROVIDER_virtual/mesa = "libgles-eabihf-dvalin-wayland-drm"
+PREFERRED_PROVIDER_virtual/mesa-gl = "libgles-eabihf-dvalin-wayland-drm"
+PREFERRED_PROVIDER_virtual/libgbm = "libgles-eabihf-dvalin-wayland-drm"
+PREFERRED_PROVIDER_virtual/egl = "libgles-eabihf-dvalin-wayland-drm"
+PREFERRED_PROVIDER_virtual/libgl = "libgles-eabihf-dvalin-wayland-drm"
+PREFERRED_PROVIDER_virtual/libgles1 = "libgles-eabihf-dvalin-wayland-drm"
+PREFERRED_PROVIDER_virtual/libgles2 = "libgles-eabihf-dvalin-wayland-drm"
+
+MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += " \
+        gpu \
+        wififw-ap6398 \
+        ap6xxx \
+        npu \
+        "
+
+RDEPENDS_packagegroup-amlogic-baserootfs_remove = " aml-subtitleserver"
+RDEPENDS_packagegroup-amlogic-baserootfs_remove = " gst-aml-drm-plugins"
+RDEPENDS_packagegroup-amlogic-baserootfs_remove = " fuse-exfat"
+RDEPENDS_packagegroup-amlogic-baserootfs_append = " npu-common"
+
+
+CHIPSET_NAME = "S905D3"
+KERNEL_DEVICETREE = "sm1_s905d3_ac200_linux.dtb"
+UBOOT_MACHINE = " sm1_ac200_v1_config"
+KERNEL_MODULE_AUTOLOAD += " dhd"
+KERNEL_MODULE_PROBECONF += "dhd"
+module_conf_dhd = "options dhd firmware_path=../../etc/wifi/6398/ nvram_path=../../etc/wifi/6398/"
+KERNEL_MODULE_AUTOLOAD += " snd-soc-dummy_codec"
+KERNEL_MODULE_AUTOLOAD += " snd-soc-aml_t9015"
+KERNEL_MODULE_AUTOLOAD += " snd-soc-tas5707"
+KERNEL_MODULE_AUTOLOAD += " snd-soc-aml_codec_tl1_acodec"
+KERNEL_MODULE_AUTOLOAD += " meson_ir"
+KERNEL_MODULE_AUTOLOAD += " hci_uart"
+KERNEL_MODULE_PROBECONF += "hci_uart"
+KERNEL_MODULE_AUTOLOAD += "meson-cpufreq"
+
+DISTRO_FEATURES_append = " drm"
+DISTRO_FEATURES_append = " gpt-partition"
+DISTRO_FEATURES_append = " kernel_515"
+
+#DISTRO_FEATURES_append = " onepass"
+DISTRO_FEATURES_append = " secure-u-boot optee "
+#DISTRO_FEATURES_append = " widevine "
+#DISTRO_FEATURES_remove = " alsa "
+DISTRO_FEATURES_append = " alsa "
+DISTRO_FEATURES_append = " pam"
+DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
+DEFAULT_DISTRO_FEATURES += " systemd"
+DISTRO_FEATURES_append = " systemd"
+DISTRO_FEATURES_remove = " sysvinit"
+DISTRO_FEATURES_append = " gstreamer1"
+DISTRO_FEATURES_append = " aml-dtv "
+DISTRO_FEATURES_append = " gst-plugin-venc"
+DISTRO_FEATURES_append = " aml-libjpeg"
+DISTRO_FEATURES_append = " aml-libvpcodec"
+DISTRO_FEATURES_append = " aml-libvphevcodec"
+#DISTRO_FEATURES_append = " playready "
+DISTRO_FEATURES_append = " softap"
+DISTRO_FEATURES_append = " aml-libgdc"
+#DISTRO_FEATURES_append = " swupdate"
+#DISTRO_FEATURES_append = " recovery"
+DISTRO_FEATURES_append = " gst-aml-imgproc"
+DISTRO_FEATURES_append = " gst-aml-gfx2d"
+DISTRO_FEATURES_append = " gst-aml-ion-allocator"
+DISTRO_FEATURES_append = " gst-jpeg-swdec"
+
+DISTRO_FEATURES_append = " gst-plugin-overlay"
+DISTRO_FEATURES_append = " gst-plugin-nn"
+DISTRO_FEATURES_append = " detect-library"
+DISTRO_FEATURES_append = " gst-plugin-conv"
+OVERRIDES .= ":ac200:sm1:onepass"
+MACHINEOVERRIDES .= ":client"
+
+EXTRA_BLUETOOTH_STUFF := "${@bb.utils.contains('DISTRO_FEATURES', 'bluetooth', bb.utils.contains('DISTRO_FEATURES', 'bluez5', ' bluez5', ' bluez4', d), '', d)}"
+EXTRA_BLUETOOTH_STUFF += " bluez-alsa"
+EXTRA_BLUETOOTH_STUFF += " brcm-patchram-plus"
+DISTRO_FEATURES_append = "${EXTRA_BLUETOOTH_STUFF}"
+
-- 
2.25.1

